docs/external-cloud-provider.md

```diff
@@ -0,0 +1,60 @@
+# External Cloud Provider
+
+To deploy a cluster using [external cloud provider](https://github.com/kubernetes-sigs/cloud-provider-openstack), create a cluster configuration with the [external cloud provider template](../../templates/cluster-template-external-cloud-provider.yaml).
+
+## Steps
+
+- After control plane is up and running, retrieve the workload cluster Kubeconfig:
+
+    ```shell
+    kubectl --namespace=default get secret/${CLUSTER_NAME}-kubeconfig -o jsonpath={.data.value} \
+    | base64 --decode \
+    > ./${CLUSTER_NAME}.kubeconfig
+    ```
+
+- Deploy a CNI solution
+
+    ```shell
+    curl https://docs.projectcalico.org/v3.12/manifests/calico.yaml | sed "s/veth_mtu:.*/veth_mtu: \"1430\"/g" | kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig apply -f -
```

Comment:
what's the 1430 for? internal cloud provider we don't have such requirements?

Reply:
MTU 1430 is [Calico MTU with IP-in-IP (IPv4)](https://docs.projectcalico.org/networking/mtu#determine-mtu-size). This size should be applied for internal cloud provider,too.

Reply:
yes, so this is not mentioned in internal cloud provider...

Reply:
> yes, so this is not mentioned in internal cloud provider...

submitted #598 for internal cloud provider